src/network/postgres_protocol_handler.cpp

```diff
@@ -1056,6 +1059,87 @@ bool PostgresProtocolHandler::ReadPacket(Buffer &rbuf, InputPacket &rpkt) {
   return true;
 }
 
+/*
+ * process_startup_packet - Processes the startup packet
+ *  (after the size field of the header).
+ */
+bool PostgresProtocolHandler::ProcessInitialPacket(InputPacket *pkt, Client client, bool ssl_able, bool& ssl_handshake, bool& finish_startup_packet) {
+  std::string token, value;
+  std::unique_ptr<OutputPacket> response(new OutputPacket());
```

Comment:
use std::make_unique<>()

https://stackoverflow.com/questions/22571202/differences-between-stdmake-unique-and-stdunique-ptr

Reply:
@tli2 Fixed. Thanks